src/PAModelTests/FilePathTests.cs

```diff
@@ -1,27 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-using Microsoft.PowerPlatform.Formulas.Tools;
-using Microsoft.PowerPlatform.Formulas.Tools.Utility;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using System.IO;
-
-namespace PAModelTests
-{
-    [TestClass]
-    public class FilePathTests
```

Comment:
>FilePathTests [](http://example.com/codeflow?start=17&length=13)

Did you intend to delete this (loss of coverage)?
Is there another test to add to fill the gap?

Reply:
the changes to FilePath were reverted so removed this one, I will add the coverage where it is currently moved to.